examples/README.adoc

```diff
@@ -0,0 +1,15 @@
+# Examples
+
+This folder contains example notebooks on how to use the `graphdatascience` python client.
+
+
+## Custom tooling for notebooks
```

Comment:
I wonder if this stuff should be under `Style guide` in `TESTING.md`?

Reply:
Which btw needs updating since changing to `ruff`

Reply:
done 🏓